pyriemann/classification.py

```diff
@@ -903,4 +902,106 @@ def predict_proba(self, X):
         prob : ndarray, shape (n_matrices, n_classes)
             Probabilities for each class.
         """
-        return softmax(-self._predict_distances(X)**2)
+        return softmax(-self._predict_distances(X) ** 2)
+
+
+def class_distinctiveness(X, y, return_num_denom=False):
```

Comment:
I would have added an argument `metric`, to generalize the function to different distances and means.
Have a look to `MDM` to see how to use `_check_metric`.
```suggestion
def class_distinctiveness(X, y, metric='riemann', return_num_denom=False):
```

Reply:
I add the argument `metric`, so please confirm it. For argument `p`, let's discuss it above.

Reply:
Great!